studio-docs/source/explorer.mdx

```diff
@@ -103,7 +103,9 @@ The Explorer currently enables you to authenticate via [request headers](#reques
 
 The bottom of the Explorer editor provides a Headers section where you can set headers that are included in your operation's HTTP request.
 
-> **Beta feature:** You can specify default headers that are applied to _every_ Explorer request executed by _every_ user in your organization. This can be useful if you want to provide a consistent identifier to your server for requests coming from the Explorer. To request access to this beta feature, please contact **support@apollographql.com**.
+#### Environment Variables
```

Comment:
"Environment variables" (by convention we use sentence casing for docs headings), same for "Default Headers" below